### ISSUE TITLE
Problems with batchsize

### ISSUE THREAD

**Issue by ljy1234-special** [nickgkan/3d_diffuser_actor#62]
May I ask if you've ever done the extent to which different batch sizes affect the results of your experiments? Or changing the batchsize, will the final converged value of the trained loss be different?

**Comment by nickgkan** [comment]
While we haven't thoroughly investigated this, we have found that on CALVIN we need a large batch size to achieve good results with the rest of the hyperparameters fixed. Larger batch size than what we used would probably be ok.

For RLBench, the batch size is not that important in our experience. You may be able to adjust other hyperparameters and get better performance.

**Comment by ljy1234-special** [comment]
Thank you！
